core/src/main/scala/org/broadinstitute/dsp/workbench/welder/StorageLinksInterp.scala

```diff
@@ -0,0 +1,17 @@
+package org.broadinstitute.dsp.workbench.welder
+
+import cats.effect.IO
+
+class StorageLinksInterp(storageLinksCache: StorageLinksCache) extends StorageLinksAlg {
+  def findStorageLink[A](localPath: RelativePath): IO[CommonContext] = for {
+    storageLinks <- storageLinksCache.get
+    baseDirectories = getPosssibleBaseDirectory(localPath.asPath)
```

Comment:
Is this ordered longest -> shortest?

Reply:
yes..see https://github.com/DataBiosphere/welder/blob/master/core/src/main/scala/org/broadinstitute/dsp/workbench/welder/package.scala#L50, the code is a bit confusing, but this might help
```
scala> 10.to(1, -1).toList
res1: List[Int] = List(10, 9, 8, 7, 6, 5, 4, 3, 2, 1)
```

`-1` is step.